styleguide/derek/view-templates/newsroom-overview/_newsroom-overview.scss

```diff
@@ -0,0 +1,149 @@
+.rsNewsroom-overview-row {
+  padding: 2rem 0;
+}
+
+.rsNewsroom-article-wrapper {
+  @include make-md-column(4);
+  @include make-sm-column(6);
+  margin: 10px 0;
+  padding: 0;
+  position: relative;
+  transition: .3s ease-in-out;
+  width: 100%;
+}
+
+.rsNewsroom-article-innerLink {
+  @include flex-box;
+  flex-flow: column nowrap;
+  height: 100%;
+  text-decoration: none;
+
+  &:hover {
+    text-decoration: none;
+  }
+}
+
+.rsNewsroom-article-logo {
+  background-image: url('../imgs/derek/rs_wordmark_white.svg');
+  background-position: 39% 50%;
+  background-repeat: no-repeat;
+  background-size: 80% 63%;
+  flex: 0 0 auto;
+  height: 30px;
+  margin: 12px 0 0 12px;
+  width: 120px;
+  z-index: 1;
+}
+
+.rsNewsroom-article-titleTopWrap {
+  flex: 1 0 auto;
+  margin: 10px 20px;
+  max-height: 150px;
+  overflow: hidden;
+  z-index: 1;
+}
+
+.rsNewsroom-article-titleTop {
+  color: $white;
+  font-size: 3.5vw;
+  font-weight: 600;
+  letter-spacing: .01vw;
+  line-height: 3.85vw;
+  margin: 0 0 10px;
+}
+
+.rsNewsroom-article-wrapperBox {
+  background: $white;
+  box-shadow: 0 1px 1px 1px $black-shadow-dark;
+
+  &:hover {
+    box-shadow: 0 15px 25px -9px $black-shadow-darkest;
+    transition: .3s ease-in-out;
+  }
+}
+
+.rsNewsroom-article {
+  height: 100%;
+}
+
+.rsNewsroom-article-top {
+  background: $gray-darkest no-repeat center center;
+  background-size: cover;
+  display: grid;
+  grid-template-rows: .25fr auto .25fr;
```

Comment:
When I built the buy now functionality, I had to make a few changes for IE support for grid.  

See: https://github.com/rackerlabs/zoolander/blob/2.0.x/styleguide/derek/global-components/modal/_buy-now.scss

I ended up just automating the conversion using this tool: https://autoprefixer.github.io/